pkg/applicationserver/grpc_deviceregistry_test.go

```diff
@@ -194,3 +200,606 @@ func TestDeviceRegistry(t *testing.T) {
 		a.So(errors.IsNotFound(err), should.BeTrue)
 	}
 }
+
+func TestDeviceRegistryGet(t *testing.T) {
+	type getByIDCallKey struct{}
+
+	registeredDevice = &ttnpb.EndDevice{
```

Comment:
This variable is not needed. Since we're "harmonizing" the tests here, please follow the structure of https://github.com/TheThingsNetwork/lorawan-stack/blob/081f5f465dce5dee7c53a40c5ee856d87e36f8e9/pkg/networkserver/grpc_deviceregistry_test.go